### PR TITLE
Add convenience `documentLoader` and zcap-context related changes.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: tests/mock-data.js [hunk 1]
```diff
@@ -25,10 +26,13 @@ mock.exampleDocWithInvocation.beta =
   require('./mock-documents/example-doc-with-beta-invocation');
 
 const didContext = require('./mock-documents/did-context');
-_loaderData['https://w3id.org/did/v0.11'] = didContext;
+_loaderData.set('https://w3id.org/did/v0.11', didContext);
```
Comment (msporny) [inline tests/mock-data.js:29]:
```suggestion
_loaderData.set('https://w3id.org/did/v1', didContext);
```

We should stop using the v0.11 context everywhere... Veres One has been upgraded to did/v1. I expect more changes will be required here.

File: tests/mock-data.js [hunk 2]
```diff
@@ -25,10 +26,13 @@ mock.exampleDocWithInvocation.beta =
...
 
 const v1Context = require('./mock-documents/veres-one-context');
-_loaderData['https://w3id.org/veres-one/v1'] = v1Context;
+_loaderData.set('https://w3id.org/veres-one/v1', v1Context);
+
+const {suiteContext} = require('@digitalbazaar/ed25519-signature-2018');
```
Comment (msporny) [inline tests/mock-data.js:34]: Are we continuing to support 2018 here?

File: tests/mock-documents/delegated-ocap-root-alpha.js [hunk 3]
```diff
@@ -1,5 +1,8 @@
 module.exports = {
-  "@context": "https://w3id.org/security/v2",
+  "@context": [
+    "https://w3id.org/zcap/v1",
+    "https://w3id.org/security/suites/ed25519-2018/v1"
```
Comment (msporny) [inline tests/mock-documents/delegated-ocap-root-alpha.js:4]: Would prefer we update to 2020 here, but understand if that's done in a separate PR.

**Review by msporny — APPROVED** [review]
Ok w/ merging, but have some non-PR related structural/dependency concerns in the comments.

File: tests/mock-data.js (hunk 1 above)
Comment (dlongley) [reply tests/mock-data.js:29]: Yeah, not doing those changes right now.

File: tests/mock-data.js (hunk 2 above)
Comment (dlongley) [reply tests/mock-data.js:34]: I don't know, I was just getting this working, it didn't actually work before with the new zcap context.

File: tests/mock-documents/delegated-ocap-root-alpha.js (hunk 3 above)
Comment (dlongley) [reply tests/mock-documents/delegated-ocap-root-alpha.js:4]: Yup, separate PR.
